eng/pipelines/common/xplat-setup.yml

```diff
@@ -49,6 +49,9 @@ jobs:
     osSubgroup: ${{ parameters.osSubgroup }}
     archType:  ${{ parameters.archType }}
 
+    librariesBuildArtifactName: ${{ format('libraries_bin_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
```

Comment:
Oopps. This is not a variable, this is defined as a parameter. It seems like you want it under the variables section.

Reply:
Thanks for doing the clean-up btw.

Reply:
Right you are, thanks for spotting, hopefully fixed now.

Reply:
Hmm, it looks like you actually sent me down a pretty nasty rathole. I'm right now trying to figure out how to consolidate the movement of buildConfig to xplat-setup with the <code>liveLibrariesBuildConfig</code> parameter but the hope to get this merged in today just keeps fading away.

Reply:
Long story short, having an independent configuration pair for CoreCLR and CoreFX is not an ideal start for a common xplat-setup script used by both.

Reply:
For now I'm inclined to roll back all the "library artifact cleanup changes" as I see no other way how to fix the pr.yml script.

Reply:
sounds good to me and sorry about that. We can do that clean-up later. 

Sorry for sending you that path, didn't think it was going to be that hard.

Reply:
No worries, it also came as a surprise to me. But yes, I think it's better to postpone the cleanup after things have settled down, we're still pretty much shooting at a moving target right now.

Reply:
Agreed.